app/models/opensearch.rb

```diff
@@ -198,4 +199,42 @@ def aggregations
       }
     }
   end
+
+  private
+
+  def match_single_field(field, match_array)
+    return unless @params[field]
+
+    match_array << {
+      match: {
+        field => @params[field].downcase
+      }
+    }
+  end
+
+  def match_single_field_nested(field, match_array)
+    return unless @params[field]
+
+    match_array << {
```

Comment:
I think a fix for that for nested fields is to use `multi_match`. I've noted that in the rebased commit message. However, it seems like this is also the case for simple fields, and I _think_ `multi_match` only works for nested...

Reply:
Yeah, I think it gets really complicated really fast and it may actually be how the fields are configured in the index so we have a bunch of directions to investigate once we get all the core bits built out.